simul.spec

```diff
@@ -0,0 +1,105 @@
+%global daos_major 0
+
+%global with_mpich 1
+%global with_openmpi3 1
```

Comment:
You will also need `with openmpi 1` for EL8.  `with_openmpi3` and `with_openmpi` are mutually exclusive where the former is for EL7 (and Leap15) and the latter is for EL8.  Please see any one of the many other tools that build openmpi and mpich variants for how to structure the addition of EL8.